### PR TITLE
[bugfix] Increases permissions checks for server routes

### PR DESCRIPTION
**Pull request by jpei**
- Must be admin and share organizations with the target user

### CODE REVIEW

File: server/api/apiController.js [hunk 1]
```diff
@@ -117,138 +118,178 @@ module.exports = {
...
   postPing: function(req, res, next) {
     var params = req.body;
-    dbUtils.getUser(params, function(error, user) {
+    dbUtils.getUser(req.user, function(error, user) {
       if (error) {
         console.error(error);
-        serveStatus(res, 500);
+        return serveStatus(res, 500);
+      }
+      if (!req.user.organization_id || req.user.organization_id !== params.organization_id) {
```
Comment (sperrow) [inline server/api/apiController.js]: I think you need to fetch this data, it's not provided by default. Example:

`req.user = { id: 9, admin_only: false }`
`params = { id: '9', visitor: '', text: '' }`

**Comment by jpei** [comment]
Updated

**Comment by jpei** [comment]
Updated

**Comment by sperrow** [comment]
Looks good

**Pull request merged**